cfgov/v1/jinja2/v1/landing-page/index.html

```diff
@@ -12,11 +12,21 @@
 {% endblock %}
 
 {% block content_main %}
+    {% set has_hero = [] %}
     {% for block in page.header -%}
         {% if block.block_type != 'hero' %}
             {{ render_block.render(block, loop.index) }}
+         {% else %}
+           {% set _ = has_hero.append(1) %}
         {% endif %}
     {%- endfor %}
+    {% if has_hero %}
```

Comment:
We could use namespaces to omit the temp variable. I would also swap the conditional around so its not checking a negation first:

```
…
{% set has_hero = namespace(found=false) %}
    {% for block in page.header -%}
        {% if block.block_type == 'hero' %}
            {% set has_hero.found = true %}
        {% else %}
            {{ render_block.render(block, loop.index) }}
        {% endif %}
    {%- endfor %}
    {% if has_hero.found %}
 …
```